Dockerfile

```diff
@@ -3,6 +3,8 @@ MAINTAINER Phuong Doan pdoan@cuahsi.org
 
 # inplaceedit in pip doesn't seem compatible with Django 1.11 yet...
 RUN pip install git+https://github.com/theromis/django-inplaceedit.git@e6fa12355defedf769a5f06edc8fc079a6e982ec
+RUN pip install \
+  markdown
```

Comment:
This has got to go!  I just cleaned this file up... put it in hs_docker_base please.

Reply:
@sblack-usu Could you point me to the line in hs_docker_base where I can add this package?

Reply:
https://github.com/hydroshare/hs_docker_base/blob/82bf23bbc02f67d0a4e7c503ed69a2e5c338e472/requirements.txt